nextflow.config

```diff
@@ -10,136 +10,157 @@
-    show_hidden_params         = false
-    schema_ignore_params       = 'genomes,modules'
-    enable_conda               = false
-    singularity_pull_docker_container = false
+    outdir                               = null
+    publish_dir_mode                     = 'copy'
+    tracedir                             = "${params.outdir}/pipeline_info"
+    email                                = null
+    email_on_fail                        = null
+    plaintext_email                      = false
+    monochrome_logs                      = false
+    help                                 = false
+    validate_params                      = true
+    show_hidden_params                   = false
+    schema_ignore_params                 = 'genomes'
+    enable_conda                         = false
 
     // Config options
-    custom_config_version      = 'master'
-    custom_config_base         = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
-    hostnames                  = [:]
-    config_profile_description = null
-    config_profile_contact     = null
-    config_profile_url         = null
-    config_profile_name        = null
+    custom_config_version                = 'master'
+    custom_config_base                   = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
+    config_profile_description           = null
+    config_profile_contact               = null
+    config_profile_url                   = null
+    config_profile_name                  = null
 
     // Max resource options
     // Defaults only, expecting to be overwritten
-    max_memory                 = '128.GB'
-    max_cpus                   = 16
-    max_time                   = '240.h'
+    max_memory                           = '128.GB'
+    max_cpus                             = 16
+    max_time                             = '240.h'
 
 }
 
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
 
-// Load modules.config for DSL2 module specific options
-includeConfig 'conf/modules.config'
-
 // Load nf-core custom profiles from different Institutions
 try {
     includeConfig "${params.custom_config_base}/nfcore_custom.config"
 } catch (Exception e) {
     System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
 }
 
-// Load igenomes.config if required
-if (!params.igenomes_ignore) {
-    includeConfig 'conf/igenomes.config'
-} else {
-    params.genomes = [:]
-}
+// Load nf-core/mag custom profiles from different institutions.
+// Warning: Uncomment only if a pipeline-specific instititutional config already exists on nf-core/configs!
+// try {
+//   includeConfig "${params.custom_config_base}/pipeline/mag.config"
+// } catch (Exception e) {
+//   System.err.println("WARNING: Could not load nf-core/config/mag profiles: ${params.custom_config_base}/pipeline/mag.config")
+// }
+
```

Comment:
I'm going to make a profile after finishing this review, as I know I will need it for my cluster and we can then uncomment this.